hooks/basic/collector.py

```diff
@@ -201,6 +266,12 @@ def settings_schema(self):
                 }
             }
         )
+        schema["Item Types UI"] = {
+            "type": "bool",
+            "default_value": False,
+            "allows_empty": True,
+            "description": "popup ui if work_path_template is none and having multiple item_types."
```

Comment:
"Popup UI to allow the user to specify the Item Type if one cannot be determined procedurally."

Reply:
updated description and given pull request